src/plugins/data/public/ui/query_editor/_query_editor.scss

```diff
@@ -86,6 +86,13 @@
   max-width: 32px;
 }
 
+// Global cancel button styling - applies everywhere
+.osdQueryEditor__cancelButton {
+  border: 1px solid #BD271E !important;
+  border-radius: 4px !important;
```

Comment:
Is the border-radius required? I don't see a difference without it and ideally we want to avoid hard coding padding. If it is required we should use EuiSizeXs.